guides/common/modules/con_rolling-content-views.adoc

```diff
@@ -7,3 +7,6 @@ You can use a rolling content view to provide a continuous stream of synchronize
 
 When you synchronize repositories to {Project}, all rolling content views that contain them get automatically updated to include the latest changes.
 You do not have to publish and/or promote a rolling content view compared to content views or composite content views.
+
+Instead, you can assign rolling content views to one or multiple lifecycle environments.
+This allows you to control which content is synchronized from {ProjectServer} to {SmartProxyServers}.
```

Comment:
Not sure if we usually make opinionated recommendations, but I feel like the following is true and could be added here:
```
We recommend using an environment other than "Library" for your rolling content views.
```
The explanation for this recommendation, is that Library already contains all latest synced content as part of the "Default Organization View". Adding your rolling CVs to Library as well necessarily means adding identical duplicate repos to Library, and by extension means syncing this content to smart proxy multiple times. Since the cost of creating/choosing an environment other than Library for your rolling CVs is negligible, it is worthwhile doing so, to avoid the duplicate smart proxy syncs.